.buildkite/hooks/pre-command

```diff
@@ -2,16 +2,15 @@
 
 set -euo pipefail
 
-
-DOCKER_REGISTRY_SECRET_PATH="kv/ci-shared/platform-ingest/docker_registry_prod"
-
-if [[ "$BUILDKITE_PIPELINE_SLUG" == "elastic-agent-package" ]]; then
```

Comment:
Why we are removing this?
It would be good to provide the docker credentials for only the pipelines and steps that actually require them.